tutorials/tour/_posts/2017-02-13-basics.md

```diff
@@ -160,6 +165,16 @@ println("Hello, " + name + "!")
 
 There are some other differences, but for now, you can think of them as something similar to functions.
 
+Method can have multi-line expressions as well.
+```tut
+def getSquareString(input: Double): String = {
+  val square = input * input
+  square.toString
+}
+```
+You can use the `return` but it is not necessary or idiomatic. Expressions
```

Comment:
I would downplay and discourage `return` more strongly. Perhaps: "The last expression in the body is the method's return value. (Scala does have a `return` keyword, but it's rarely used.)"